.github/workflows/specs.yml

```diff
@@ -3,24 +3,22 @@ on: [push, pull_request]
 jobs:
   tests:
     strategy:
+      fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        ruby: [2.7]
     runs-on: ${{ matrix.os }}
     steps:
       - uses: ruby/setup-ruby@v1
         with:
-          ruby-version: ${{ matrix.ruby }}
```

Comment:
I think doing this in matrix makes sense, as we could test different versions of ruby...

Reply:
Do the tests even use the current version of Ruby?
One important thing to note here is the `ruby` in PATH is not a Ruby built or installed by RVM.
I would think for tests you would want to test a RVM Ruby.
I can remove this change anyway.

Reply:
Yeah, you're probably right!